_includes/docs_menu.html

```diff
@@ -88,10 +88,10 @@
         </li>
         <li class="nav-header">Reference</li>
         <li>
-            <a href="{{ site.baseurl }}/docs/index.html" target="_blank">API Reference</a>
+            <a href="{{ site.baseurl }}/docs/index.html">API Reference</a>
         </li>
         <li>
-            <a href="{{ site.baseurl }}/docs/trex_tableau_viz_ref.html" target="_blank">Tableau Viz Reference</a>
+            <a href="{{ site.baseurl }}/docs/trex_tableau_viz_ref.html">Tableau Viz Reference</a>
```

Comment:
My mistake. The VizAPI isn't a separate doc set and so it shouldn't have the `target="_blank"`